src/modules/http-client/implementation/rate-limiter-middleware.js

```diff
@@ -0,0 +1,11 @@
+const rateLimiter = require('express-rate-limit');
+const { SERVICE_API_RATE_LIMIT } = require('../../../constants/constants');
+
+module.exports = () =>
+    rateLimiter({
+        windowMs: SERVICE_API_RATE_LIMIT.TIME_WINDOW_MILLS,
```

Comment:
Can you move rate limiting constants to configuration so we can easily control the numbers?